mkdocs.yml

```diff
@@ -54,8 +56,21 @@ markdown_extensions:
   - pymdownx.emoji
   - pymdownx.keys
   - pymdownx.superfences
+  - abbr
+  - attr_list
+  - pymdownx.snippets:
+      auto_append:
+        - snippets/abbreviations.md
```

Comment:
Would it be possible to have this folder and file within the `docs` folder instead of at the root of the project ?
It feels pretty weird to have it there...

Reply:
For this specific use case no ; I've seen in the documentation that it has to be an external file and confirmed it with a test. I wanted to see what you would say about this because I find it useful to see what an abbreviation means without having to open another page. But that would mean we keep the abbreviations.md outside.

Reply:
I believe it should be possible. Have you tried moving the snippets folder in docs and updating the `mkdocs.yml` file with:

```yaml
...
- pymdownx.snippets:
      base_path: docs
      auto_append:
          - snippets/abbreviations.md
```

?

Reply:
That being said, I totally agree on the fact that it is very useful to get the meaning of abbreviations without having to open another page. It is a great suggestion ! 👍

Reply:
I tried with :
``` markdown
- pymdownx.snippets:
      auto_append:
          - abbreviations.md
```
, the file `abbreviations.md` being in the docs folder. But I can try as you suggested also !

Reply:
Well your solution works so that's perfect ! We can also get read of the markdown-include extension and make full use of snippets instead :-)